config/default/kustomization.yaml

```diff
@@ -75,81 +87,157 @@ replacements:
+      index: 1
+    select:
+      kind: Certificate
+      name: serving-cert
+      namespace: kserve
+  - fieldPaths:
+    - metadata.annotations.[cert-manager.io/inject-ca-from]
+    options:
+      delimiter: '/'
+      index: 0
+    select:
+      kind: CustomResourceDefinition
+      name: inferenceservices.serving.kserve.io
+  - fieldPaths:
+    - metadata.annotations.[cert-manager.io/inject-ca-from]
+    options:
+      delimiter: '/'
+      index: 0
+    select:
+      kind: MutatingWebhookConfiguration
+      name: inferenceservice.serving.kserve.io
+  - fieldPaths:
+    - metadata.annotations.[cert-manager.io/inject-ca-from]
+    options:
+      delimiter: '/'
+      index: 0
+    select:
+      kind: ValidatingWebhookConfiguration
+      name: inferenceservice.serving.kserve.io
+  - fieldPaths:
+    - metadata.annotations.[cert-manager.io/inject-ca-from]
+    options:
+      delimiter: '/'
+      index: 0
+    select:
+      kind: ValidatingWebhookConfiguration
+      name: trainedmodel.serving.kserve.io
+  - fieldPaths:
+    - metadata.annotations.[cert-manager.io/inject-ca-from]
+    options:
+      delimiter: '/'
+      index: 0
+    select:
+      kind: ValidatingWebhookConfiguration
+      name: inferencegraph.serving.kserve.io
+  - fieldPaths:
+    - metadata.annotations.[cert-manager.io/inject-ca-from]
+    options:
+      delimiter: '/'
+      index: 0
+    select:
+      kind: ValidatingWebhookConfiguration
+      name: clusterservingruntime.serving.kserve.io
+  - fieldPaths:
+    - metadata.annotations.[cert-manager.io/inject-ca-from]
+    options:
+      delimiter: '/'
+      index: 0
     select:
       kind: ValidatingWebhookConfiguration
       name: servingruntime.serving.kserve.io
+  - fieldPaths:
+      - metadata.annotations.[cert-manager.io/inject-ca-from]
+    options:
+      delimiter: '/'
+      index: 0
+    select:
+      kind: ValidatingWebhookConfiguration
+      name: localmodelcache.serving.kserve.io
 
-# Protect the /metrics endpoint by putting it behind auth.
-# Only one of manager_auth_proxy_patch.yaml and
-# manager_prometheus_metrics_patch.yaml should be enabled.
-# If you want your controller-manager to expose the /metrics
-# endpoint w/o any authn/z, uncomment the following line and
-# comment manager_auth_proxy_patch.yaml.
+    # Protect the /metrics endpoint by putting it behind auth.
+    # Only one of manager_auth_proxy_patch.yaml and
+    # manager_prometheus_metrics_patch.yaml should be enabled.
+    # If you want your controller-manager to expose the /metrics
+    # endpoint w/o any authn/z, uncomment the following line and
+    # comment manager_auth_proxy_patch.yaml.
+    # Only one of manager_auth_proxy_patch.yaml and
+    # manager_prometheus_metrics_patch.yaml should be enabled.
+    #- manager_prometheus_metrics_patch.yaml
 patches:
 - path: manager_image_patch.yaml
-#- path: manager_auth_proxy_patch.yaml
+# FIXME(upstream): auth_proxy_overlay?
+# - path: manager_auth_proxy_patch.yaml
```

Comment:
so upstream comment this out, right? What is the FIXME action item?

Reply:
No, this is the only thing we have to comment out. But I think we can fix it upstream with an overlay (maybe)

https://github.com/kserve/kserve/blob/b5e3d533eb7e2f5b6a02b9d4c01a6baa00fc0eb0/config/default/kustomization.yaml#L205

Alternatively we can unpatch it in the overlay, but I would consider this as a follow-up. WDYT?

I commented earlier https://github.com/opendatahub-io/kserve/pull/815/files#diff-c006b1ef995eeeb57408cce42bd84db8dfc0cbe18df03222857665cebb1ac6fd

Reply:
We can do this way. I have no issues with it